firmware/src/network_model.c

```diff
@@ -333,14 +290,11 @@ int NetworkModel_SetDHCPMode(const RDMHeader *header,
 
 int NetworkModel_GetZeroconfMode(const RDMHeader *header,
                                  const uint8_t *param_data) {
-  unsigned int index = 0;
```

Comment:
I'm not following.

The return-if makes things tricky. I don't like introducing macros.